php/composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "minimum-stability": "stable",
+
+    "require": {
+        "hubspot/api-client": "*",
```

Comment:
Should we fix the specific api-client version?

Reply:
I  think we shouldn't because it is fix in composer.lock